proto/ord_schema.proto

```diff
@@ -783,3 +823,19 @@ message Percentage {
   // Precision of the measurement (with the same units as `value`).
   float precision = 2;
 }
+
+message Selectivity {
```

Comment:
I think you forgot to keep a ```Selectivity selectivity``` field in the reaction outcome. I do like the idea of separating this out

Reply:
Good catch; fixed.